src/Makefile.am

```diff
@@ -88,7 +88,7 @@ nodist_bin_SCRIPTS = gpgrt-config
 m4datadir = $(datadir)/aclocal
 m4data_DATA = gpg-error.m4 gpgrt.m4
 
-pkgconfigdir = $(datadir)/pkgconfig
+pkgconfigdir = $(libdir)/pkgconfig
```

Comment:
installing pkg-config files at libdir enables multilib support, for example lib and lib64 with different settings. using datadir is incorrect for native libraries.